src/Kernel32.Desktop/Kernel32.cs

```diff
@@ -4,9 +4,8 @@
 namespace PInvoke
 {
     using System;
-    using System.IO;
     using System.Runtime.InteropServices;
-    using FILETIME = System.Runtime.InteropServices.ComTypes.FILETIME;
```

Comment:
The problem with the other `FILETIME` occurs within the file in the shared project, the file on the desktop project doesn't reference `FILETIME` at all and both R# and the default analyzers are suggesting to remove it.

Also while the R# one can be suppressed it doesn't seem possible to suppress the Roslyn suggestion.

If we keep it I think that it should come of a comment indicating why it should be kept here.